### PR TITLE
I've updated 6 commits on Readme, heroes and history/mine

### PR DESCRIPTION
**Pull request by josediazvarela**

### CODE REVIEW

**Comment by josediazvarela** [comment]
It seems we don't have conflicts with your Github Principal? Pls, check it out!!!!

File: heroes.md [hunk 1]
```diff
@@ -6,4 +6,5 @@
 * Aquaman
 * Mujer Maravilla
 * Linterna verde
+* Mazzinger Z
```
Comment (joseluisdiaz9) [inline heroes.md]: Why do you add mazzinger z!!!

Comment (josediazvarela) [reply heroes.md]: This change out to be here. but if you want i can remove it

Comment (joseluisdiaz9) [reply heroes.md]: Yes, Remove it !

**Review by joseluisdiaz9 — CHANGES_REQUESTED** [review]
Jose Diaz Varela changes that fucking _**Mazinger Z**_!!! -- remove it ...
@josediazvarela

**Comment by joseluisdiaz9** [comment]
JoseLuisDiaz9 I've done the changes requested!!!

**Comment by josediazvarela** [comment]
I 've done the changes requested 
@joseluisdiaz9

**Review by joseluisdiaz9 — APPROVED** [review]

**Comment by joseluisdiaz9** [comment]
I am gonna put all commits or changes in the Repository Principal. i did create and merge commit

**Pull request merged**